sdk/nodejs/runtime/resource.ts

```diff
@@ -209,7 +209,10 @@ export function getResource(
             });
         }),
         label,
-    );
+    ).catch(() => {
+        // Request to the engine is likely due to another failure in the program and another error
```

Comment:
Wonder if it make sense to print of stderr or something here?

Reply:
I added logging, but it looks awkward so I removed it. You can catch and print the exception yourself now.

recording of the experience with `console.error()`:
https://asciinema.org/a/IqkuoVgu97biyiSpdfcef2vVe